src/Domain.LinnApps/GoodsIn/GoodsInService.cs

```diff
@@ -150,7 +150,7 @@ public class GoodsInService : IGoodsInService
                 transactionType,
                 createdBy,
                 partNumber,
-                qty,
+                total,
```

Comment:
I think this was the culprit @andrewfraser73. I was only passing the do_bookin routine the qty of one of the lines as opposed to the total. Not sure why this fixes it but it seems to

Reply:
maybe the qty passed capped off the amount it did but if it sorts it then that goods